### DISCUSSION TITLE
is there a docker image for play 2.3.8 with java?

### DISCUSSION THREAD

**Discussion by michael-luk** [playframework/playframework#11697] [General]
Im a play developer using 2.3.8 an old version. now looking for a docker image to speed up the deployment of my projects.

**Comment by mkurz** [comment]
I don't think there are ready to go Play docker images. However you can take a look at
* https://hub.docker.com/r/hseeberger/scala-sbt
* https://hub.docker.com/r/sbtscala/scala-sbt

these are docker images containing sbt, so you should be able to work with that.

If you want to package your application into a docker image: Play already comes with sbt-native-packager, docs can be found here: https://www.playframework.com/documentation/2.3.x/ProductionDist#The-Native-Packager

However I highly recommend to upgrade to latest Play 2.8.x...
